examples/favorites_advanced/getx_favorites_advanced/lib/feature_puppy/controllers/puppy_list_controller.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+
+import 'package:favorites_advanced_base/repositories.dart';
+import 'package:favorites_advanced_base/models.dart';
+
+class PuppyListController extends GetxController {
+  PuppyListController(){
+    _repository = Get.find<PuppiesRepository>();
```

Comment:
The same as above. It's recommended converting this simplicity dependency to an implicit one

Reply:
Corrected

Reply:
@RomanovaPrime, my expectation is that PuppyListController will have PuppiesRepository as an argument